actors/evm/src/lib.rs

```diff
@@ -51,12 +51,16 @@ fn test_method_selector() {
 #[repr(u64)]
 pub enum Method {
     Constructor = METHOD_CONSTRUCTOR,
-    // TODO: Do we want to use ExportedNums for all of these, per FRC-42?
     InvokeContract = 2,
     GetBytecode = 3,
     GetStorageAt = 4,
     InvokeContractDelegate = 5,
     GetBytecodeHash = 6,
+    // FRC-42 method exports
+    InvokeContractExported = frc42_dispatch::method_hash!("InvokeContract"),
+    InvokeContractDelegateExported = frc42_dispatch::method_hash!("InvokeContractDelegate"),
+    GetBytecodeExported = frc42_dispatch::method_hash!("GetBytecode"),
+    GetBytecodeHashExported = frc42_dispatch::method_hash!("GetBytecodeHash"),
```

Comment:
This probably shouldn't be exported.

Reply:
Getting the keccak hash of the bytecode is a highly public no?

Reply:
See my comment above.